### DISCUSSION TITLE
AppImage Fedora 41

### DISCUSSION THREAD

**Discussion by docile-ux** [OrcaSlicer/OrcaSlicer#7544] [Q&A]
Hello,

I have a problem with OrcaSlicer. I downloaded the software, but I can't launch it, even though I selected the "run as executable" option.

Can you help me solve this problem? Thanks in advance for your assistance

**Comment by madheader** [comment]
The AppImage seems to rely on the deprecated webkit2gtk4.0. 
Would it be possible to switch to the 'new' webkit2gtk4.1?
